go/api/scripting_and_function_commands_test.go

```diff
@@ -854,3 +854,134 @@ func ExampleGlideClusterClient_FunctionListWithRoute() {
 	//   Engine: LUA
 	//   Functions: 1
 }
+
+func ExampleGlideClient_FunctionDump() {
+	client := getExampleGlideClient()
+
+	// Call FunctionDump to get the serialized payload of all loaded libraries
+	_, err := client.FunctionDump()
+	if err != nil {
+		fmt.Println("Error:", err.Error())
+	} else {
+		fmt.Println("Function dump got a payload")
```

Comment:
~Are you sure that this test always runs after a function load?~ dump doesn't fail if there are not functions.

We probably could do a test with empty output. It will validate that function doesn't fail. Alternative option: replace `else` by `if len(dump) > 0`.

You are ok to keep that as is